wordpress-mobile/react-native-aztec
# Feature/match wpandroid api level

This PR:

* Downgrades the `targetSdkVersion` to 26 to match wpandroid's in preparation to integrating the mobile Gutenberg editor into wpandroid
* Cleans up the unused resources to make wpandroid's linter happy

### To test (A):
Run the example app and it should work as normal.

### To test (B):
Use the https://github.com/wordpress-mobile/WordPress-Android/tree/feature/integrate-gutenberg wpandroid branch to run wpandroid and open a GB post. That branch is currenctly configured to use https://github.com/wordpress-mobile/gutenberg-mobile/pull/201, which in turn uses this version of the `react-native-aztec` wrapper. It should work as normal.
